### PR TITLE
Create doc for aviatrix_transit_vpc resource

### PR DESCRIPTION
**Pull request by srinitt**

### CODE REVIEW

File: website/docs/r/aviatrix_transit_vpc.html.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The following arguments are supported:
+
+* `cloud_type` - (Required) Type of cloud service provider, requires an integer value. Use 1 for AWS
+* `account_name` - (Required) This parameter represents the name of a Cloud-Account in Aviatrix controller.
+* `gw_name` - (Required) Name of the gateway which is going to be created.
+* `vpc_id` - (Required) VPC-ID/VNet-Name of cloud provider.  Example: AWS: "vpc-abcd1234", ARM: "VNet_Name:Resource_Group_Name", GCP: "mygooglecloudvpcname", etc...
+* `vpc_reg` - (Required) Region of cloud provider. Example: AWS: "us-east-1", ARM: "East US 2", etc...
+* `vpc_size` - (Required) Size of the gateway instance.  Example: AWS: "t2.large", etc...
+* `subnet` - (Required) Public Subnet INFO.  Example: AWS: "CIDR~~ZONE~~SubnetName", ARM: "CIDR~~SubnetName", GCP: "CIDR~~ZONE~~VpcName", etc...
```
Comment (arvindsree) [inline website/docs/r/aviatrix_transit_vpc.html.markdown]: Is this subnet name or the CIDR of the subnet?

File: website/docs/r/aviatrix_transit_vpc.html.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* `dns_server` - (Optional) Specify the DNS IP
```
Comment (arvindsree) [inline website/docs/r/aviatrix_transit_vpc.html.markdown]: You can add that this is only required while using a custom private DNS for the VPC

File: website/docs/r/aviatrix_transit_vpc.html.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* `ha_subnet` - (Optional) HA Subnet
+* `tag_list` - (Optional) Instance tag of cloud provider Example: key1:value1,key002:value002
```
Comment (arvindsree) [inline website/docs/r/aviatrix_transit_vpc.html.markdown]: Looking at the code, this seems to be a list of strings. Probably [ "key1:value1", "key2:value"] ? Needs to be tested

File: website/docs/r/aviatrix_transit_vpc.html.markdown (hunk 1 above)
Comment (arvindsree) [reply website/docs/r/aviatrix_transit_vpc.html.markdown]: We can use "10.0.0.0/16~~us-east1c~~subnet1" as an example which is clearer

File: website/docs/r/aviatrix_transit_vpc.html.markdown (hunk 2 above)
Comment (srinitt) [reply website/docs/r/aviatrix_transit_vpc.html.markdown]: Makes sense, thanks for the suggestion

File: website/docs/r/aviatrix_transit_vpc.html.markdown (hunk 3 above)
Comment (srinitt) [reply website/docs/r/aviatrix_transit_vpc.html.markdown]: You are right. I got carried away while copying from the API

**Pull request merged**